pkg/detectors/box/box.go

```diff
@@ -0,0 +1,138 @@
+package box
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"box", "access"}) + `\b([0-9a-zA-Z]{32})\b`)
```

Comment:
why "access"? seems a bit generic

Reply:
No idea, why I put "access". seems like I put it for testing. Let me remove it.

Reply:
No idea, why I put "access". seems like I put it for testing. Let me remove it.